### ISSUE TITLE
NTR: GMCSF high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8342]
New term request GMCSF high in blood (elthat_130528041621): Granulocyte-macrophage colony-stimulating factor; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000245
- label: Elevated circulating granulocyte-macrophage colony-stimulating factor concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:15378942
- def: The concentration of granulocyte-macrophage colony-stimulating factor (GM-CSF) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8342
- synonyms: Elevated circulating GM-CSF concentration(NONE)|GMCSF high in blood(NONE)
